lib/html/d3-graph.js

```diff
@@ -287,11 +296,43 @@
   function onZoomChanged() {
     graph.attr("transform", "translate(" + d3.event.translate + ")" + " scale(" + d3.event.scale + ")");
   }
+    
+  function persistNodes(nodes) {
+    var store = nodes.filter(function(n){ return n.fixed }).reduce(function(mem,n){
+      mem[n.id]=n;
+      return mem;
+    },{});
+    store = JSON.stringify(store);
+    window.localStorage.setItem("dependo"+window.graphIdentification, store);
+    console.log(store); //for manual saving
```

Comment:
That's actually a feature...
I can open console and copy the last entry, then put it in the html file in a field that's already prepared there.
We can iterate to make it nicer, but for now it's usable.